design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,658 @@
+  # or
+  managementPolicy: ["*"]
+
+# ObserveOnly
+spec:
+  managmentPolicy: ObserveOnly
+
+# would be replaced with:
+spec:
+  managementPolicy: ["Observe"]
+
+# OrphanOnDelete
+spec:
+  managementPolicy: OrphanOnDelete
+
+# would be replaced with:
+spec:
+  managementPolicy: ["Create", "Update", "Observe", "LateInitialize"]
+
+# Ignore changes - turn off late init and just have fields that are managed
+# by Crossplane in spec.forProvider
+spec:
+  managementPolicy: ["Create", "Update", "Delete", "Observe"]
+```
+
+In addition to the new management policy, we will also add a new field
+`spec.initProvider` which would contain parameters that should only be
+used at the creation time. This would allow users to specify fields that
+should be ignored on updates, but still be used during creation.
+
+## Implementation
+
+### Management policy
+
+The new management policy would be implemented in the `crossplane-runtime`.
+Most of the work will be in the [Reconciler], where we will need to add
+management policy checks to sections of code where we create, update etc.
+
+For example:
+- Create - [Create Section]
+- Update - [Update Section]
+- Delete - [Delete Section]
+- Late Init - [Late Init Section]
+- Observe - this one is tricky because we still need to observe/get a resource,
+  so we just need to make sure that we don't update `status.atProvider`.
+
+### Deletion policy
+
+`deletionPolicy` was planned to be deprecated in favour of the new management
+policies according to [the ObserveOnly design doc.][ObserveOnly], but still
+retain some functionality if a non-default value was set. In practice, it
+meant that if the `deletionPolicy` was set to `Orphan`, and the
+`managementPolicy` set to `FullControl`, the external resource would be
+orphaned.
+
+In the new design, we could still follow this approach, by orphaning the
+resource even if the `managementPolicy` includes `Delete`, if the
+`deletionPolicy` is set to `Orphan`.
+
+### ["Create", "Update", "Delete", "Observe", "LateInitialize"] or ["*"] by default
+
+Except for the cosmetic value, this decision affects how future additions to
+the management policy would be handled. If we go with the `["*"]` approach,
+every new management policy would be automatically included in the resources,
+which may be what we want. Similarly, if we go with the `["Create", "Update",
+"Delete", "Observe", "LateInitialize"]` approach, every new management policy
+would need to be added manually or through a migration.
+
+So this is something to keep in mind when developing new management policies.
+For this proposal, we will go with the `["*"]` approach as its more
+future-proof.
+
+### Migrating existing resources
+
+The `managementPolicy` feature is alpha, so it should be ok to change the
+API. We should still offer a migration path for existing resources. The
+combinations of `managementPolicy` and `deletionPolicy` should be migrated
+to the new `managementPolicy` field.
```

Comment:
Renamed the new field to `managementPolicies`. 

We cant just change the field type as it would remove the existing data in the field, and although the ObserveOnly feature is alpha, its probably used by a bunch of people due to it being a very sought after feature.

So the plan would be to keep the deprecated `managementPolicy` field for a release, and provide a migration tool if we think its necessary.